generators/app/templates/_index.html

```diff
@@ -20,7 +20,7 @@
   <script>
     (function(window, videojs) {
       var player = window.player = videojs('<%= pluginName %>-player');
-      player.<%= pluginFunctionName %>();
+      var <%= pluginFunctionName %> = window.<%= pluginFunctionName %> = player.<%= pluginFunctionName %>();
```

Comment:
Added this small fix in here, as I think its nice to be able to use you plugin on window right away.